composer.json

```diff
@@ -15,10 +15,7 @@
       "Shopsys\\MigrationBundle\\": "packages/migrations/src/",
       "Shopsys\\Plugin\\": "packages/plugin-interface/src/",
       "Shopsys\\ProductFeed\\": "packages/product-feed-interface/src/",
-      "Shopsys\\ProductFeed\\ZboziBundle\\": "packages/product-feed-zbozi/src/",
```

Comment:
First, Im not sure why this was neccessary. Removal of all feeds? Im propably not comfortable with the solution - remove all and create new. This should be the last option if everything fails. Not the first one. With this approach, everyone can remove all while refactoring.

Reply:
No, no, no, you've got this wrong - it's only for the time being - I will drop this commit and refactor the feeds. I only didn't want to deal with them during the work on a single feed.

After the review I will drop the commit and redo the rest of the feeds just like I did with the Google feed.

Reply:
Thank you for clearing up